subscriptionform/templates/registration/password_reset_done.html

```diff
@@ -1,8 +1,9 @@
 {% extends "base.html" %}
+{% load i18n %}
 
 {% block content %}
-    <p>Please check your email and follow the instructions to reset your password.</p>
-    <p>If you don't receive an email, please make sure you've entered the address you registered
-    with and check your spam folder.</p>
+    <p>{% trans 'Please check your email and follow the instructions to reset your password.' %}</p>
+    <p>{% blocktrans %} If you do not receive an email, please make sure you have entered the address
```

Comment:
dit zou je ook dus in een wat grotere blocktrans kunnen gooien. In een blocktrans kan je ook simpele html tags gebruiken.